sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -290,6 +289,32 @@ private[ui] class ExecutionPagedTable(
         }
       }
 
+    var headerNameWithTooltips: Map[String, String] = Map("ID" -> "Execution ID of the SQL query.",
+      "Description" ->
```

Comment:
I think  we can follow the same approach as #25723 , by a seperate ToolTips class? If @dongjoon-hyun agrees then no issues. Also not sure, for every headers we need to give tool tips.

Reply:
@shahidki31, (Can we have a simple map here instead of getHeaderTooltips and ToolTips.scala? Currently, it seems too much for tooltips.) - this is suggested by @dongjoon-hyun , so I think no need to implement again.

Reply:
you can check above comments of @dongjoon-hyun.

Reply:
@shahidki31 . That's too much for this.

Reply:
Thanks @dongjoon-hyun